QRScanner/QRScannerView.swift

```diff
@@ -202,6 +202,8 @@ public class QRScannerView: UIView {
     }
 
     private func moveImageViews(qrCode: String, corners: [CGPoint]) {
+        assert(Thread.isMainThread)
```

Comment:
Add assertion (now this is OK because this method is called in `DispatchQueue.main.async{}`)